README.md

```diff
@@ -499,7 +499,8 @@ Declare parameters on methods and constructors with either `ParameterSpec.builde
 `FunSpec`'s convenient `addParameter()` API:
 
 ```kotlin
-val android = ParameterSpec.builder("android", String::class)
+val android = ParameterSpec.builder("android", String::class.asTypeName().asNullable())
```

Comment:
Why make it nullable? For the readme I'd prefer to avoid the noise considering it doesn't change the behavior.

Reply:
That was in response to an issue asking about how to add nullable parameters. Since there isn't an example in the readme for nullable parameters, I thought this might be a good place to exhibit both the default value and nullable. 

I'm open to adding it elsewhere.

Reply:
Please do. Just a short description & a fun example!